src/app/shared/dockstore.model.ts

```diff
@@ -69,6 +69,7 @@ export class Dockstore {
   static GITHUB_APP_INSTALLATION_URL = 'will be filled in by configuration.service';
 
   static DOCUMENTATION_URL = 'https://docs.dockstore.org';
+  static FEATURED_CONTENT_URL = 'will be filled in by configuration.service';
```

Comment:
Make sure to update the compose setup with this new variable.

Reply:
Was merged yesterday 👍